deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccountName: csi-snapshotter
       containers:
         - name: csi-provisioner
-          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
```

Comment:
Let's wait for the upcoming csi-provisioner release which depends on snapshotter 6.0.1

Reply:
sure , we can wait

Reply:
@xing-yang updated to v3.2.0 ... :+1: